lib/usecases/validators.js

```diff
@@ -55,7 +55,7 @@ async function meetsEligibilityCriteria({
     tradingInHackney === true &&
     VALID_BUSINESS_ID.includes(typeOfBusinessId) &&
     tradingOn20200311 === true &&
-    servedLegalNotices === true &&
+    servedLegalNotices === false &&
```

Comment:
I knew I would get one of these wrong 👍

Reply:
I think it was correct, then I changed it 😂